### ISSUE TITLE
Create and train dataset by scrapping websites?

### ISSUE THREAD

**Issue by saileshkush95** [gunthercox/ChatterBot#2101]
Can I use this package to automatically create and train dataset by scrapping whole website.  Suppose I just develop chat bot using this package. Now I want to  create and train dataset, once I integrate in any website.

**Comment by AnAnalogGuy** [comment]
> Can I use this package to automatically create and train dataset by scrapping whole website
No. And it wouldn't make sense as you should train the bot with some questions-answers structured dialogs. Best approach to train a bot always is to use dialogs from the actual or a very similar use case.
